apps/BasicExample/android/app/build.gradle

```diff
@@ -45,7 +45,7 @@ react {
 
     /* Hermes Commands */
     //   The hermes compiler command to run. By default it is 'hermesc'
-    // hermesCommand = "$rootDir/my-custom-hermesc/bin/hermesc"
+    hermesCommand = "../../node_modules/react-native/sdks/hermesc/osx-bin/hermesc"
```

Comment:
Why only two levels up here, as opposed to four above?

Reply:
I was following [this tutorial] of setting RN app inside monorepo. However, it looks like that this path doesn't make any sense since `react-native` directory is not present in pointed `node_modules` (at least not at this point). I've changed it in [512fc8e](https://github.com/software-mansion/react-native-gesture-handler/pull/3490/commits/512fc8e6dd66fe69e3d8f28647814a0c69628e92)